com.unity.render-pipelines.high-definition/Editor/Sky/HDLightingWindowEnvironmentSection.cs

```diff
@@ -208,6 +208,8 @@ void UpdateSkyIntPopupData()
             VolumeProfile profile = m_SerializedActiveSceneLightingSky.volumeProfile;
             if (profile != null)
             {
+                bool foundID = false;
```

Comment:
What is this change?

Reply:
Because when testing, I noticed that if you select a sky for the static lighting and then you delete it, the UI shows a blank dropdown, so it's just a small fix